.github/workflows/helm_charts_scalar.yml

```diff
@@ -12,42 +12,68 @@ on:
     paths:
       - "charts/**"
 
+env:
+  HELM_VERSION: v3.5.2
+
 jobs:
   lint-chart:
     runs-on: ubuntu-latest
+    env:
+      PYTHON_VERSION: 3.7
+
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Do we need to fetch all history for all tags and branches here?

Reply:
I'm not sure.

I referred to the [chart-testing-action](https://github.com/helm/chart-testing-action/blob/main/README.md) and the previous code also used fetch with `--unshallow`.

Reply:
Not fetching all the history will speed up the CI, but it is a small repository at this point, so the impact is negligible.